packages/webapp/public/locales/es/translation.json

```diff
@@ -856,10 +856,15 @@
       "AMPHIBIANS": "Anfibio",
       "BIRDS": "Aves",
       "CROP_VARIETIES": "Variedades de cultivos",
-      "ERROR": {
+      "API_OFFSET_LIMIT_ERROR": {
+        "BODY": "Los límites de su granja son demasiado amplios, no podemos proveerle un reporte de Biodiversidad. Intente crear límites más pequeños.",
+        "PREVIEW": "Unavailable",
```

Comment:
"PREVIEW" is in English in all files :)

Reply:
I think this key is actually not being used anywhere, maybe I should just go ahead and delete it?